src/components/Header/Nav/Links/Link.js

```diff
@@ -1,22 +1,34 @@
 // @flow
 import styled from 'react-emotion';
 
-export default styled('a')({
+import palette from 'pasteup/palette';
+import { textSans } from 'pasteup/fonts';
+import { tablet, desktop } from 'pasteup/breakpoints';
+
+export default styled('a')(props => ({
     fontSize: 14,
-    fontFamily:
-        '"Guardian Text Sans Web","Helvetica Neue","Helvetica","Arial","Lucida Grande","sans-serif"',
-    color: '#121212',
+    fontFamily: textSans,
+    color: palette.neutral['1'],
     float: 'left',
     lineHeight: 1.2,
     position: 'relative',
     transition: 'color 80ms ease-out',
     padding: '6px 10px',
     margin: '1px 0 0',
     textDecoration: 'none',
+    display: 'none',
     ':hover': {
         textDecoration: 'underline',
     },
     ':focus': {
         textDecoration: 'underline',
     },
-});
+    ...(props.index < 2 && {
```

Comment:
👋 `nth-child`

Reply:
Hmm.... What is this doing?

Reply:
Can we call this prop `nthChild`? Would that be more obvious?

Reply:
hmmm, maybe a `hideUntilDesktop` prop?

Reply:
Yeah that could work. No big deal though, I think I was just temporarily thrown by the name!

Reply:
like this https://github.com/guardian/guui/pull/38

it's intersting it wasn't clear to you, would be good to alight on something more transparent